druid/src/widget/widget_ext.rs

```diff
@@ -79,6 +79,17 @@ pub trait WidgetExt<T: Data>: Widget<T> + Sized + 'static {
         SizedBox::new(self).height(height)
     }
 
+    /// Wrap a widget in an [`Align`] and a [`SizedBox`] to make it a fixed size
+    ///
+    /// [`Align`]: struct.Align.html
+    /// [`SizedBox`]: struct.SizedBox.html
+    fn sized(self, width: f64, height: f64) -> Align<T> {
+        SizedBox::new(self)
+            .width(width)
+            .height(height)
+            .align_vertical(UnitPoint::CENTER)
```

Comment:
I would prefer that we make the user explicitly call this. One of the goals of `WidgetExt` is to make it very easy to write out these sorts of compositions, and my current inclination is to try to avoid hidden composition where possible.